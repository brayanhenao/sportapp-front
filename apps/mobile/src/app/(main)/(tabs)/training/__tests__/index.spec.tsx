import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'

import Training from '../training'

describe('Training', () => {
	let component: ReactTestRenderer

	beforeEach(() => {
		component = renderer.create(<Training />)
	})

	afterEach(() => {
		component.unmount()
	})

	it('should render correctly', () => {
		expect(component.toJSON()).toMatchSnapshot()
	})
})
