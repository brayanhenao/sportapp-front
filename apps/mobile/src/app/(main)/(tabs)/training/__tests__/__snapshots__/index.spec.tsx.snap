// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Training should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello From Trainig
  </Text>
</View>
`;
